tests/jballerina-unit-test/src/test/resources/test-src/expressions/mappingconstructor/spread_op_field.bal

```diff
@@ -159,6 +159,9 @@ function testSpreadFieldWithRecordTypeHavingNeverField() {
     Grades grades = { physics: 75 };
     Address address= { street: "Main Street" };
 
+    record {| string street; |} addressInline = {...address};
+    assertEquality("Main Street", addressInline.street);
```

Comment:
Shall we add a test where we assign `()` to the never type field. like below 
```
record {|
        int id;
        string name;
        never dept?;
    |} rec = {
        name: "Joy",
        id: 1023,
        dept: ()
    };
```

Reply:
Add below test as well

```
type Employee record {|
    int id;
    string name;
    string dept;
|};

public function dummyTest() {
    record {|
        int id;
        string name;
        never dept?;
    |} rec = {
        name: "Joy",
        id: 1023,
        dept: ()
    };
    Employee _ = {dept:"a", ...rec};
}

```

Reply:
Added